.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
We should add our IDE settings (`.vscode`) to `.gitignore`.
Then you would also need to remove tracking of this file - [more info](https://stackoverflow.com/a/1274447).